keps/sig-cli/2227-kubectl-default-container/README.md

```diff
@@ -310,12 +344,19 @@ resource usage (CPU, RAM, disk, IO, ...) in any components?**
 
 ## Implementation History
 
-2021-03-10: Alpha implementation completion in 1.21
+2021-03-10(v1.21) Alpha: implementation completion
 - https://github.com/kubernetes/kubernetes/pull/97099
 - https://github.com/kubernetes/kubernetes/pull/99615
 - https://github.com/kubernetes/kubernetes/pull/99581
 - https://github.com/kubernetes/kubernetes/pull/99833
 
+2021-10-28(v1.24): make 'kubectl logs' default to the first container when default container cannot be 
```

Comment:
Maybe instead of 1.23, we can use this point as beta, in that case (ie. update other comments where I'm talking about beta accordingly).

Reply:
Updated.